src/main/java/frc/robot/Robot.java

```diff
@@ -160,7 +160,7 @@ public void teleopPeriodic() {
         limelight.setLightEnabled(false);
       }
 
-      drive.arcadeDrive(turnInput, speedInput);
+      drive.arcadeDrive(turnInput, speedInput, driver.getAButtonPressed());
```

Comment:
I think this is the best place to call the boost code (hopefully as a function). Rather than pass whether to boost down two or more layers of function calls, we can just adjust turnInput and speedInput here instead.

Reply:
implemented